### PR TITLE
Minicylc fix 2

### LINKED ISSUE #72: minicylc: fix for non-firefox browsers
See https://github.com/cylc/cylc-sphinx-extensions/pull/71#pullrequestreview-1051288750

### PR DESCRIPTION
**Pull request by oliver-sanders**
Closes #72 

Low priority, I just wanted this working for the presentation.

A somewhat nasty solution, but it seems to work where others don't.

### CODE REVIEW

File: cylc/sphinx_ext/minicylc/_static/js/minicylc.js [hunk 1]
```diff
@@ -18,8 +18,17 @@
...
 var minicylc_default_theme = {
+    'waiting_fill': 'none',
+    'waiting_stroke': 'black',
+    'running_fill': '#55c3e5ff',
+    'running_stroke': 'black',
+    'succeed_fill': '#64c77eff',
+    'succeed_stroke': 'black'
+}
+
+// Default Cylc colour theme.
```
Comment (oliver-sanders) [inline cylc/sphinx_ext/minicylc/_static/js/minicylc.js:30]:
```suggestion
```

**Comment by MetRonnie** [comment]
I'm getting the same error while building the docs that we're getting on GH Actions, despite not having changed my `SQLAlchemy`/`ERAlchemy` versions for months

**Comment by oliver-sanders** [comment]
This project doesn't use ERAlchemy so it should work from this end.

**Review by MetRonnie — APPROVED** [review]
tested

**Review by MetRonnie — APPROVED** [review]

**Pull request merged**